### DISCUSSION TITLE
Latest release 4.10.3: does anyone notice a "Just a test" in the Responses section

### DISCUSSION THREAD

**Discussion by nickn-github** [swagger-api/swagger-ui#7973] [General]
Latest release 4.10.3: does anyone notice a "Just a test" in the Responses section

I am just checking to see if this should be there

**Comment by mike-loux-planview** [comment]
Using Swagger in an ASP.NET Core 6 web api project via Swashbuckle.  
I see this as well, but only when I enable the `requestSnippetsEnabled` option.  
The extra snippets are great; the `Just a test` message, not so much.  
Would be nice to be able to configure that specifically to say something else (or nothing at all).
